src/main/java/nextstep/converter/MemberConverter.java

```diff
@@ -0,0 +1,15 @@
+package nextstep.converter;
+
+import nextstep.auth.application.dto.AuthMember;
+import nextstep.member.domain.Member;
+
+public class MemberConverter {
+
+    private MemberConverter () {
+        throw new UnsupportedOperationException("Utility class");
+    }
+
+    public static AuthMember MemberToAuthMember(Member member) {
```

Comment:
메서드 이름은 소문자로 시작하도록 작성해보면 어떨까요?
```suggestion
    public static AuthMember memberToAuthMember(Member member) {
```